types/plotly.js/index.d.ts

```diff
@@ -348,8 +356,8 @@ export function prependTraces(
     update: Data | Data[],
     indices: number | number[],
 ): Promise<PlotlyHTMLElement>;
-export function toImage(root: Root, opts: ToImgopts): Promise<string>;
-export function downloadImage(root: Root, opts: DownloadImgopts): Promise<string>;
+export function toImage(root: RootOrData, opts: ToImgopts): Promise<string>;
```

Comment:
```suggestion
export function toImage(root: RootOrData, opts?: ToImgopts): Promise<string>;
```
https://github.com/plotly/plotly.js/blob/v2.6.4/src/plot_api/to_image.js#L81